src/members/index.js

```diff
@@ -1,80 +1,33 @@
-                "email": "Cyril_Pollich16@hotmail.com"
-            },
-            {
-                "id": "21d764d7-542f-4a81-abc1-0a7b162a76a7",
-                "name": "Quincy Donnelly",
-                "avatar": "https://cdn.fakercloud.com/avatars/namankreative_128.jpg",
-                "phone": "328-976-3872 x125",
-                "email": "Quincy32@hotmail.com"
-            },
-            {
-                "id": "3810273b-48e6-478c-a6bc-7a1171de18b8",
-                "name": "Glenna Schaden",
-                "avatar": "https://cdn.fakercloud.com/avatars/gregsqueeb_128.jpg",
-                "phone": "1-600-465-4093 x35770",
-                "email": "Glenna.Schaden6@gmail.com"
-            },
-            {
-                "id": "0a055f39-9d6c-4e85-a6dc-400a1212baef",
-                "name": "Harold Gutmann",
-                "avatar": "https://cdn.fakercloud.com/avatars/maximsorokin_128.jpg",
-                "phone": "1-268-572-8174",
-                "email": "Harold10@gmail.com"
-            },
-            {
-                "id": "ba121cb1-9ee9-4c33-b7e0-abe6fedf9efc",
-                "name": "Janis Wyman",
-                "avatar": "https://cdn.fakercloud.com/avatars/amandabuzard_128.jpg",
-                "phone": "(389) 962-7951",
-                "email": "Janis_Wyman@hotmail.com"
-            },
-            {
-                "id": "35e88c7c-e61c-4aa9-add0-d62ac9d1e2c6",
-                "name": "Maye Larkin",
-                "avatar": "https://cdn.fakercloud.com/avatars/ooomz_128.jpg",
-                "phone": "(692) 425-8538",
-                "email": "Maye79@gmail.com"
-            },
-            {
-                "id": "2f43a096-cfa7-4e6f-ae4f-a59f70449bb9",
-                "name": "Arvid Kshlerin",
-                "avatar": "https://cdn.fakercloud.com/avatars/mashaaaaal_128.jpg",
-                "phone": "500-660-8480",
-                "email": "Arvid.Kshlerin@yahoo.com"
-            },
-            {
-                "id": "c59eb7c1-9277-437d-a5cb-8cbceb94a87d",
-                "name": "Zena O'Hara",
-                "avatar": "https://cdn.fakercloud.com/avatars/begreative_128.jpg",
-                "phone": "954.275.7375",
-                "email": "Zena.OHara@gmail.com"
-            },
-        ];
+    console.log('members index user', user)
+
+    useEffect(() => {
+        const url='https://us-central1-hanbit-directory-dev.cloudfunctions.net/v1/members';
+        const options = {
+            headers: {
+                Authorization: `Bearer ${user.accessToken}`
+            }
+        }
+        console.log('options', options)
+        axios.get(url, options)
+            .then((response) => {
+                // handle success
+                setMembers(response.data);
+            })
+            .catch((error) => {
+                // handle error
+                console.log(error);
+            })
+            .finally(() => {
+                // always executed
+            });
```

Comment:
@byum1996 Move this code into `api` layer so that useEffect has to just call a function, instead of using axios directly.

Reply:
you should see an example in the sangstagram code base.